gcs/CONFIGURATION.md

```diff
@@ -514,6 +514,64 @@ permissions (not authorized) to execute these requests.
     Minimum size in bytes of the read range for Cloud Storage request when
     opening a new stream to read an object.
 
+### grpc-directpath configuration
+
+gRPC over directpath is an optimized configuration to connect with gcs backend.
+It offers better latency and increase bandwidth.
+GCS-connector enables this path for IO operations (for now). It's no a default
```

Comment:
reword "GCS-connector enables this path for IO operations (for now)" to. Currently supported only for read/write operations?